x/examples/outline-connectivity-app/shared_frontend/pages/connectivity_test/index.ts

```diff
@@ -811,9 +813,56 @@ export class ConnectivityTestPage extends LitElement {
 
             <dt class="results-list-item-data-key">${msg("Time")}</dt>
             <dd class="results-list-item-data-value">${result.durationMs}ms</dd>
+
+            ${isSuccess
+              ? nothing
+              : html`
+                  <dt class="results-list-item-data-key">${msg("Error")}</dt>
+                  <dd class="results-list-item-data-value">${result.error?.message}</dd>
+                `}
           </dl>
         </li>`;
       })}
     </ul>`;
   }
 }
+
+@customElement('info-popup')
+export class InfoPopup extends LitElement {
+  @property({ type: Boolean }) isPopupVisible = false;
+  @property({ type: String }) popupText = 'This is an info pop-up. Click the button to toggle visibility.'
+
+  static styles = css`
+  .popup {
+    display: none;
+    position: absolute;
+    z-index: 1000; /* A high value to ensure it's on top */
```

Comment:
@daniellacosse Thanks for the feedback. I got `popover` working on browser and iOS and desktop app but on Android it does not function as intended. Please see the screen shots below for Android and iOS. I suspect the WebView on Android (emulator pixel 7, API level 33) maybe using a Chrome version older than 114 which does not support `popover`. I am still investigating that theory. Please also let me know if you have any suggestions on how to best position the `popover` box on the screen.

iOS:
<img width="282" alt="Screenshot 2023-12-12 at 12 27 19 PM" src="https://github.com/Jigsaw-Code/outline-sdk/assets/117060873/e56a0594-95af-498a-80fb-dc224fe5b576">

Android:
<img width="282" alt="Screenshot 2023-12-11 at 9 50 01 PM" src="https://github.com/Jigsaw-Code/outline-sdk/assets/117060873/f917cbca-f251-4a17-bd96-b1e132ff0bf7">

Reply:
I managed to get popover working on all platforms; Please `android` and `ios` screenshots below. There is a bit of behavior different between popover on web/desktop and ios/android; On web, clicking outside of the popover area hides it but on `ios` and `android` nothing happens. On all platforms, clicking/tapping on close button `x`, will hide the popover.

Reply:
I managed to get `popover` working on all platforms. 

The issue on Android was due to the fact that Capacitor uses WebView which is not automatically updated on Emulator. Once I logged into Google Play on Emulator and upgrade WebView manually it fixed the issue. I think this should work fine on most recent Android devices but If the device runs a very old version of Android API / WebView, the popover element won't work as expected. 

Also, I observed that on iOS tapping outside of the `popover` window does not hide it as other platforms. This is not a big deal since I added a close button `x` to the popover window to offer an alternative to close it down. 

<img width="264" alt="Screenshot 2023-12-25 at 6 45 52 PM" src="https://github.com/Jigsaw-Code/outline-sdk/assets/117060873/cf591214-e0a6-4c25-9747-f8e9b7bb41d0">
<img width="264" alt="Screenshot 2023-12-25 at 8 21 34 PM" src="https://github.com/Jigsaw-Code/outline-sdk/assets/117060873/99a74121-f7a8-4600-8c94-766b491e3d9b">